docs/src/main/sphinx/security/certificate.rst

```diff
@@ -0,0 +1,108 @@
+==========================
+Certificate authentication
+==========================
+
+You can configure Trino to support client-provided certificates validated by the
+Trino server on initial connection.
+
+.. important::
+
+    This authentication method is only provided to support sites that have an
+    absolute requirement for client authentication *and already have* client
+    certificates for each client. Sites in this category have an existing PKI
+    infrastructure, possibly including an onsite Certificate Authority.
+
+    This feature is not appropriate for sites that would need to generate a set
+    of client certificates in order to use this authentication method. Consider
+    instead using another :doc:`authentication method
+    </security/authentication-types>`.
+
+Using certificate authentication
+--------------------------------
+
+All clients connecting with HTTPS/TLS go through the following initial steps:
+
+* The client attempts to contact the coordinator.
+* The coordinator returns its certificate to the client.
+* The client validates the server's certificate using the client's trust store.
+
+A cluster with certificate authentication enabled goes through the following
+additional steps:
+
+* The coordinator asks the client for its certificate.
+* The client responds with its certificate.
+* The coordinator verifies the client's certificate, using the coordinator's
+  trust store.
+
+Several rules emerge from these steps:
+
+1. Trust stores used by clients must include the certificate of the signer of
```

Comment:
I dont think these rules should be numbered .. there is no order to them right?

Reply:
Done